lib/blueberry_rails/app_builder.rb

```diff
@@ -175,17 +174,15 @@ def configure_simple_form
 
     def replace_users_factory
       remove_file 'spec/factories/users.rb'
-      template 'users_factory.rb', 'spec/factories/users.rb'
+      copy_file 'users_factory.rb', 'spec/factories/users.rb'
     end
 
     def replace_root_controller_spec
       remove_file 'spec/controllers/root_controller_spec.rb'
-      template 'root_controller_spec.rb', 'spec/controllers/root_controller_spec.rb'
+      copy_file 'root_controller_spec.rb', 'spec/controllers/root_controller_spec.rb'
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.